go/service/chat_local.go

```diff
@@ -59,12 +59,98 @@ func (h *chatLocalHandler) GetThreadLocal(ctx context.Context, arg keybase1.GetT
 	return h.unboxThread(ctx, boxed, arg.ConversationID)
 }
 
+func retryUpToNTimesUntilNoError(n int, action func() error) (err error) {
```

Comment:
OK to make the intention of this clear (in case we need one with timed wait in the future), I'll change this function's name to `retryUpToNTimesUntilNoErrorWithoutBackoff`